doc/README.md

```diff
@@ -39,43 +39,51 @@ Tracker](https://issues.jenkins-ci.org/issues/?jql=project+%3D+JENKINS+AND+compo
 1.  Configure your project
 2.  Select: Build -\> Add build step -\> execute shell/Execute Windows
     batch command
-3.  Add the command to run the tests. You might have direct pybot/jybot
-    command here like "pybot --variable name:value --outputdir
-    my\_robot\_results my\_test\_suite" or in case you need to do
+3.  Add the command to run the tests. You might have direct robot/jybot
+    command here like `robot --variable name:value --outputdir
+    my_robot_results my_test_suite` or in case you need to do
     something more, you can call any script that you use for running the
     tests. Relevant part from Robot Jenkins Plugin point of view is that
     this script generates the Robot outputs.
-4.  Force your Robot script to return successfully from shell with "exit
-    0" to empower the plugin in deciding if the build is success/failure
+4.  Force your Robot script to return successfully from shell with `exit
+    0` to empower the plugin in deciding if the build is success/failure
     (by default Robot exits with error code when there's any failed
-    tests)
+    tests). You can do this for example by adding `--nostatusrc` to your
```

Comment:
Link `--nostatusrc` to relevant part in User Guide: http://robotframework.org/robotframework/latest/RobotFrameworkUserGuide.html#return-codes